packages/story-editor/src/components/fontCheck/fontCheckDialog.js

```diff
@@ -0,0 +1,196 @@
+
+const Spacer = styled.div`
+  margin-top: 22px !important;
+`;
+
+const ReversedIcon = styled(Icons.ChevronRightSmall)`
+  transform: rotate(180deg);
+`;
+
+const DashboardLink = styled.span`
+  display: flex;
+  color: black;
+  align-self: flex-start;
+  align-items: center;
+  font-weight: normal;
+  margin-right: 40px;
+`;
+
+export const FontCheckDialog = ({ isOpen, onClose, missingFont }) => {
+  const { dashboardLink, dashboardSettingsLink } = useConfig();
+  const defaultFont = useMemo(() => {
+    return {
+      font: {
+        id: TEXT_ELEMENT_DEFAULT_FONT?.family,
+        ...TEXT_ELEMENT_DEFAULT_FONT,
+      },
+    };
+  }, []);
+  const [suggestedFont, setSuggestedFont] = useState(defaultFont);
+  const { updateElementsByFontFamily } = useStory((state) => ({
+    updateElementsByFontFamily: state.actions.updateElementsByFontFamily,
+  }));
+
+  /**
+   * Handles updating suggested font which is passed to the FontPicker
+   *
+   * @param {Object} [evt] Font object.
+   * @return void
+   */
+  const onPushUpdate = useCallback((evt) => {
+    if (evt?.font) {
+      setSuggestedFont({
+        font: { id: evt.font.family, family: evt.font.family },
+      });
+    }
+  }, []);
+
+  const onSettingsClick = useCallback((evt) => {
+    trackClick(evt, 'click_video_optimization_settings');
+  }, []);
+
+  const updateMissingFontWithDefault = useCallback(() => {
+    updateElementsByFontFamily({
+      family: missingFont,
+      properties: { font: defaultFont.font },
+    });
+    onClose();
+  }, [defaultFont, onClose, missingFont, updateElementsByFontFamily]);
+
+  const updateMissingFontWithSelected = useCallback(() => {
+    updateElementsByFontFamily({
+      family: missingFont,
+      properties: { font: suggestedFont.font },
+    });
+    onClose();
+  }, [suggestedFont, onClose, missingFont, updateElementsByFontFamily]);
+
+  // @todo Should we enforce a button click vs ESC etc... closing the dialog ?
+  return (
+    <Dialog
+      isOpen={isOpen}
+      onClose={onClose}
+      title={__('Missing Fonts', 'web-stories')}
+      contentLabel={__('Missing Fonts', 'web-stories')}
+      actions={
+        <>
+          <Link href={dashboardLink}>
+            <DashboardLink>
+              <ReversedIcon width="32" height="32" />
+              {__('Back to dashboard', 'web-stories')}
+            </DashboardLink>
+          </Link>
+          <Button
+            type={BUTTON_TYPES.SECONDARY}
+            size={BUTTON_SIZES.SMALL}
+            onClick={updateMissingFontWithDefault}
+          >
+            {__('Open anyway', 'web-stories')}
+          </Button>
+          <Button
+            type={BUTTON_TYPES.PRIMARY}
+            size={BUTTON_SIZES.SMALL}
+            onClick={updateMissingFontWithSelected}
+          >
+            {__('Replace font', 'web-stories')}
+          </Button>
+        </>
+      }
+      primaryText={__('Replace Font', 'web-stories')}
+    >
+      <Spacer>
```

Comment:
if you wanted to, instead of a div for space you could just have a container div and set up `display: grid` with a gutter. Achieve the same thing and you don't need to use `!important`.

Reply:
Good call that's a much better idea :)